functions.go

```diff
@@ -450,52 +455,44 @@ func (f *Function) CreateInstanceSnapshot() {
 	if err != nil {
 		log.Panic(err)
 	}
-
-	snap := snapshotting.NewSnapshot(f.vmID, "/fccd/snapshots", f.imageName)
-	err = orch.CreateSnapshot(ctx, f.vmID, snap)
+	
+	snap, err := f.snapshotManager.InitSnapshot(f.fID, f.imageName)
 	if err != nil {
 		log.Panic(err)
 	}
-
-	_, err = orch.ResumeVM(ctx, f.vmID)
+	
```

Comment:
clean up. don't forget to lint

Reply:
Actually, I guess it is safer to resume the VM here (though it is not used after this point and is subject to shut down).

> clean up

What do you mean?

> don't forget to lint

What is the preferred way to lint? I am not aware of a way to lint only the diffed code, while linting all the files I change would bloat the diff with a lot of formatting changes.

Reply:
use goland or vim-pliugin, for example. all of them should yield the same output (in my experience)

Reply:
Added a separate commit for this.